### ISSUE TITLE
Explorer.exe Restarts When the Search Window Is Closed

### ISSUE THREAD

**Issue by Seuong** [stnkl/EverythingToolbar#225]
**Describe the bug**

When the EverythingToolbar UI closes, either from opening a file or clicking away, explorer.exe restarts itself. Searching for items works as expected.  Looking into the .dmp of explorer.exe shows that "SearchBox.xaml.cs" was causing the "problem". Removing the line "HistoryManager.Instance.AddToHistory(EverythingSearch.Instance.SearchTerm);"  and building fixes the issue for me. 

**Steps to reproduce**

Crashes when using versions > 0.6.3. I would assume other people would report this, so I guess it's something to do with my setup. I updated Windows, tried different graphics drivers, installed a different version of Everything, and tried opening the toolbar without Everything running. None of the these things seemed to work.

**Expected behavior**

EverythingToolbar UI closes without restarting explorer.exe.

<details>
<summary>Log output</summary>

```
2022-02-11 01:31:29.5363|INFO|EverythingToolbar started. Version: 0.7.4.0, OS: Microsoft Windows NT 10.0.19044.0|
2022-02-11 01:31:29.6831|INFO|Everything version: 1.4.1|
```
</details>

**Comment by Horst12** [comment]
With toolbar version 0.7.4.0 no such problems.
______________________________________________________
Windows 11 Home x64 Version 21H2 (OS Build 22000.493)
Everything 1.5.0.1300a (x64)

**Comment by Seuong** [comment]
I experienced the issue with all versions above 0.6.3, including 0.7.4, I suppose it's something specific to my computer. No problems though, I'll try to look into a bit more.

**Comment by stnkl** [comment]
Sorry for the late reply. Were you able to track down the issue further?

**Comment by Seuong** [comment]
Unfortunately no... I think it may have had to do with my GPU, but couldn't say for sure. Sorry I couldn't help :/.

**Comment by micahmo** [comment]
+1 I have this exact issue. Unfortunate that this was closed. 😞

**Comment by stnkl** [comment]
Hi @micahmo, which version of EverythingToolbar are you using?

**Comment by micahmo** [comment]
Hey @stnkl, thanks for the super fast response! I was not expecting anything considering the lack of activity on this issue. 😊 I'm running version `0.7.4.0`. Let me know if there's any other info I can gather.

**Comment by stnkl** [comment]
I was just working on EverythingToolbar anyway. 😊

Could you please try updating to the latest beta release? If I remember correctly I might have fixed this issue already there, but I can't remember for sure.

https://github.com/stnkl/EverythingToolbar/releases/tag/0.8.0-beta2

**Comment by micahmo** [comment]
I am using `winget` to install, and the latest version available there is `0.8.0.1`, which I assume corresponds to `0.8.0-beta1`. However, that did the trick! Everything is working now. Thank you so much for the help!

**Comment by stnkl** [comment]
Awesome! I will also have to get the winget thing sorted out at some point. But at least there is an open ticket for that. Happy it works now!
